### DISCUSSION TITLE
Define aHrefs bots as crawlers

### DISCUSSION THREAD

**Discussion by sujanbasnet1** [vercel/next.js#60679] [Ideas]
### Goals

1. Define a new bot as a crawler - [Ahrefs](https://ahrefs.com/).

### Background

[aHrefs](https://ahrefs.com/) is a SEO audit tool.

`fallback:true` works as `fallback:blocking` for crawlers. aHrefs is an active web crawler (calimed by them) that does not see this behavior, leading to incorrect reporting. Current alternative is to set `fallback: blocking` for everyone.

### Proposal

Add aHrefs bot to the list of crawlers.

Contributing? - Yes, definitely, zero on open-source contributions but I was looking at that PR that added this feature, looks like is just adding the bot name in a regex.

aHrefs has two bots - [AHrefsBot](https://ahrefs.com/robot) and [AHrefsSiteAudit](https://ahrefs.com/robot/site-audit)

**Comment by kevludwig** [comment]
Is this the reason some pages (absolutely random) run into timeouts with error code 0?

**Reply by hvess** [reply]
I am wondering about this also. I am seeing a lot of very random timeouts when auditing a site with Ahrefs.

**Reply by kevludwig** [reply]
![image](https://github.com/user-attachments/assets/3cab5f91-6b42-41be-a941-c00950f3caee)

I've noticed this behavior over the last few days, especially with applications hosted on Vercel. Today, a message appeared saying that there are issues with AWS Shield, which could also maybe affect Vercel?

**Reply by hvess** [reply]
I think it's very likely related, considering Vercel is built on top of AWS. I am experiencing the issue on all of Vercel hosted applications as well my AWS hosted application.

**Reply by murillo94** [reply]
If you navigate to `/firewall/configure` in the Vercel project dashboard, there's a **System Bypass Rules** section (_"Configure the IP addresses and CIDRs that should bypass Vercel's system mitigations."_) but it's only available for Pro members. 😢  

Is anyone with a Pro account able to test it and share the results?

<img width="1308" alt="Screenshot 2025-02-06 at 18 42 00" src="https://github.com/user-attachments/assets/72f6fd3d-fbdb-4d11-96c1-2136b4f4c5ad" />

**Reply by kevludwig** [reply]
> If you navigate to `/firewall/configure` in the Vercel project dashboard, there's a **System Bypass Rules** section (_"Configure the IP addresses and CIDRs that should bypass Vercel's system mitigations."_) but it's only available for Pro members. 😢  
> 
> 
> 
> Is anyone with a Pro account able to test it and share the results?
> 
> 
> 
> <img width="1308" alt="Screenshot 2025-02-06 at 18 42 00" src="https://github.com/user-attachments/assets/72f6fd3d-fbdb-4d11-96c1-2136b4f4c5ad" />

Yes, I successfully configured the firewall. Scan is working now.
